File: src/main/java/org/jesperancinha/rockstarts/rockstarsmanager/repository/ArtistsRepository.java
package org.jesperancinha.rockstarts.rockstarsmanager.repository;

import org.jesperancinha.rockstarts.rockstarsmanager.model.Artist;
import org.springframework.data.repository.CrudRepository;

public interface ArtistsRepository extends CrudRepository<Artist, Long> {
    Artist findArtistsByName(String name);
}

File: src/main/kotlin/org/jesperancinha/rockstarts/rockstarsmanager/repository/ArtistsRepository.kt
package org.jesperancinha.rockstarts.rockstarsmanager.repository

import org.jesperancinha.rockstarts.rockstarsmanager.model.Artist
import org.springframework.data.repository.CrudRepository

interface ArtistsRepository : CrudRepository<Artist, Long> {
    fun findArtistsByName(name: String): Artist
}